crates/aptos/src/node/local_testnet/utils.rs

```diff
@@ -31,10 +32,51 @@ pub fn get_docker() -> Result<Docker> {
 
 pub async fn confirm_docker_available() -> Result<()> {
     let docker = get_docker()?;
-    docker
+    let info = docker
         .info()
         .await
         .context("Docker is not available, confirm it is installed and running. On Linux you may need to use sudo.")?;
+
+    info!("Docker Info: {:?}", info);
+
+    let version = docker
+        .version()
+        .await
+        .context("Failed to get Docker version")?;
+
+    info!("Docker Version: {:?}", version);
+
+    // Try to warn the user about their Docker version being too old. We don't error
+    // out if the version is too old in case we're wrong about the minimum version
+    // for their particular system. We just print a warning.
+    match version.api_version {
+        Some(current_api_version) => match Version::from(&current_api_version) {
+            Some(current_api_version) => {
+                let minimum_api_version = Version::from("1.42").unwrap();
```

Comment:
Curious, why did we set 1.42 as the min version?

Reply:
Semi arbitrary, this is just a somewhat recent version. Larry and Maayan had a really old version of Docker and that's why it didn't work for them.